config/routes.rb

```diff
@@ -7,7 +7,7 @@
   root 'users#index'
```

Comment:
- [ ] Kindly remove the commented line of code on line 6. This will make your codebase look clean and professional. You can always use git to revisit the code if need be.